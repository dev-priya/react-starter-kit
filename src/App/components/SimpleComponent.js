import React from 'react';
import PropTypes from 'prop-types';

const SimpleComponent = ({ message }) => <div>{message}</div>;

SimpleComponent.propTypes = {
  message: PropTypes.string.isRequired
};

export default SimpleComponent;