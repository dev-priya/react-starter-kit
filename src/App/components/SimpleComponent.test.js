import React from 'react';
import {shallow} from 'enzyme';
import SimpleComponent from './SimpleComponent';

describe('SimpleComponent', () => {
  const wrapper = shallow(<SimpleComponent message="test message" />);

  it('renders a div', () => {
    expect(wrapper.find('div').length).toBe(1);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});