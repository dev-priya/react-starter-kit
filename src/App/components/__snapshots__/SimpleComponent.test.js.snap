// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleComponent matches the snapshot 1`] = `
<div>
  test message
</div>
`;
